fetch.bs

```diff
@@ -2643,7 +2647,10 @@ boolean <var>http3Only</var>, run these steps:
  <li><p>If a secure transport is used, <var>timingInfo</var>'s
```

Comment:
The above paragraphs already talk about secure connections implicitly, so maybe some of this should be moved up?

For HTTP/3, I'm not sure you need to add anything, though if you wanted to say "If a secure transport is used, such as TLS over TCP or QUIC, ..." that could clarify that you're including both.

Reply:
Gotcha. The main important addition is that in QUICK `secure connection start` and `connection start` are identical. Perhaps I should just add that and forget the rest.

Reply:
Thanks, saying that explicitly makes tons of sense to me.